client/executor/src/integration_tests/mod.rs

```diff
@@ -811,7 +811,7 @@ fn return_max_memory_offset(wasm_method: WasmExecutionMethod) {
 
 	assert_eq!(
 		call_in_wasm("test_return_max_memory_offset", &[], wasm_method, &mut ext).unwrap(),
-		().encode()
+		vec![0]
```

Comment:
Maybe actually it would be better if you set some value at this memory position to ensure it actually works.

Reply:
Makes sense, I had thought about maybe doing that because I wasn't sure if it was guaranteed to always be `0`. Done.